src/test/java/org/heigit/ohsome/ohsomeapi/controller/DataExtractionTest.java

```diff
@@ -142,20 +139,20 @@ public void elementsCentroidTest() {
         .get("geometry").get("coordinates").size());
   }
 
-  // this needs a fix in the OSHDB to work
-  // see https://github.com/GIScience/oshdb/issues/338
-  //  @Test
-  //  public void elementsClipGeometryParamTrueFalseTest() {
-  //    TestRestTemplate restTemplate = new TestRestTemplate();
-  //    String uri = "/elements/geometry?bboxes=8.700582,49.4143039,8.701247,49.414994&time=2018-01-02"
-  //        + "&filter=building=* and (geometry:other or geometry:line)";
-  //    ResponseEntity<JsonNode> emptyFeatureResponse =
-  //        restTemplate.getForEntity(server + port + uri + "&clipGeometry=false", JsonNode.class);
-  //    ResponseEntity<JsonNode> featureResponse =
-  //        restTemplate.getForEntity(server + port + uri + "&clipGeometry=true", JsonNode.class);
-  //    assertTrue(emptyFeatureResponse.getBody().get("features").size() == 0);
-  //    assertTrue(featureResponse.getBody().get("features").size() == 1);
-  //  }
+  @Test
+  public void elementsClipGeometryParamTrueFalseTest() {
+    TestRestTemplate restTemplate = new TestRestTemplate();
+    String uri = "/elements/geometry?bboxes=8.700582,49.4143039,8.701247,49.414994&time=2018-01-02"
+        + "&filter=building=* and (geometry:other or geometry:line)";
+    ResponseEntity<JsonNode> clipGeometryFalseResponse =
+        restTemplate.getForEntity(server + port + uri + "&clipGeometry=false", JsonNode.class);
+    ResponseEntity<JsonNode> clipGeometryTrueResponse =
+        restTemplate.getForEntity(server + port + uri + "&clipGeometry=true", JsonNode.class);
+    assertEquals("Polygon", clipGeometryFalseResponse.getBody().get("features").get(0)
+        .get("geometry").get("type").asText());
+    assertEquals("GeometryCollection", clipGeometryTrueResponse.getBody().get("features").get(0)
+        .get("geometry").get("type").asText());
```

Comment:
If I got it correctly, you changed the check from checking if the response is or isn't empty (size is 0 or 1) to checking the type of the first Feature. Wouldn't it make sense to check both or is this redundant?